### ISSUE TITLE
Cannot dropdown on the top-right corner

### ISSUE THREAD

**Issue by loongmxbt** [nicolaslopezj/orion-admin-only-bootstrap#1]
The account name dropdown cannot use. Maybe because not include the bootstrap js files.

**Comment by frabrunelle** [comment]
+1

It looks like that for me:

![screen shot 2015-08-27 at 2 17 44 pm](https://cloud.githubusercontent.com/assets/1114844/9529101/6bac505c-4cc6-11e5-93a4-134d142598c5.png)

**Comment by lukelarsen** [comment]
I have this problem as well.

**Comment by lukelarsen** [comment]
I have figured a band-aid solution to this. What the project is missing is the bootstrap js. I installed bootstrap with 'meteor add twbs:bootstrap'. Then I viewed the tag until I found the bootstrap.js file. I made a copy of it and then uninstalled bootstrap, 'meteor remove twbs:bootstrap'. Once I loaded the bootstrap js copy into my meteor app this works again and I am able to keep the bootstrap styles out of my client facing app. The problem with this solution is that now the bootstrap js is being loaded on both the client and admin sides of the app.
